examples/getting_started/example_save_and_load.py

```diff
@@ -111,11 +138,24 @@ def prepare_ocp(
     print(f"Final objective value : {np.nansum(sol.cost)} \n")
     sol.print()
 
-    # --- Save the optimal control program and the solution --- #
+    # --- Save the optimal control program and the solution with stand_alone = False --- #
     ocp.save(sol, "pendulum.bo")  # you don't have to specify the extension ".bo"
 
     # --- Load the optimal control program and the solution --- #
     ocp_load, sol_load = OptimalControlProgram.load("pendulum.bo")
 
     # --- Show results --- #
     sol_load.animate()
+    sol.graphs()
```

Comment:
sol, should be `sol_load` otherwise you don't plot the loaded solution, but the solved solution which is irrelevant for this example

Reply:
Done